rotkehlchen/rotkehlchen.py

```diff
@@ -1076,8 +1076,8 @@ def get_settings(self, cursor: 'DBCursor') -> DBSettings:
         """Returns the db settings with a check whether premium is active or not"""
         return self.data.db.get_settings(cursor, have_premium=self.premium is not None)
 
-    def get_cache(self, cursor: 'DBCursor') -> dict[DBCache, Timestamp]:
-        return self.data.db.get_cache(cursor)
+    def get_cache(self, cursor: 'DBCursor') -> dict[str, int]:
+        return self.data.db.get_cache_for_api(cursor)
```

Comment:
For @LefterisJP : this one is just a wrapper but existed here before this PR. I believe we can remove it but I'm not sure if you allowed it for some reason

Reply:
Since you have to check the tests that failed remove this function and query rotkhelchen.data.db.get_cache_for_api. is one function call less and this is just a wrapper